### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by igorline**

### CODE REVIEW

File: 1/incrementing-my-value.md [hunk 1]
```diff
@@ -78,4 +78,4 @@ Remember to run `cargo +nightly test` to test your work.
 
 [embedded-code-previous](./assets/1.5-finished-code.rs ':include :type=code embed-previous')
 
-<!-- tabs:end -->
\ No newline at end of file
+<!-- tabs:end -->
```
Comment (Robbepop) [inline 1/incrementing-my-value.md]: what changed here?

Comment (igorline) [reply 1/incrementing-my-value.md]: @Robbepop really nothing! I was editting this file via github ui and it got changed.
Let me fix that.

**Comment by igorline** [comment]
@Robbepop, please review

**Comment by Robbepop** [comment]
> @Robbepop, please review

Thank you!

**Pull request merged**